hasjob/views/listing.py

```diff
@@ -162,13 +160,31 @@ def jobdetail(domain, hashid):
 
     is_bgroup = getbool(request.args.get('b'))
     headline = post.headlineb if is_bgroup and post.headlineb else post.headline
+    is_post_admin = is_siteadmin or post.admin_is(g.user) or (g.user and g.user.flags.is_employer_month)
```

Comment:
Yes, `lastuser` reads from the request context, so you can use it outside the view. (Specifically, it reads from `g.userinfo`). Function overheads aren't significant unless you're calling one in a loop, where you get `O(n)` performance degradation. In this case it's just `O(1)` so that's fine.